src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.ui.xml

```diff
@@ -45,9 +45,10 @@
          <rw:FormListBox ui:field="listPresetFilePatterns_"
                     styleName="{style.presetFilePatterns}">
             <g:item value="">All Files</g:item>
-            <g:item value="*.r, *.R, *.rnw, *.Rnw, *.rmd, *.Rmd, *.rmarkdown, *.Rmarkdown, *.rhtml, *.Rhtml, *.rd, *.Rd, *.h, *.hpp, *.c, *.cpp">Common R source files (R, C/C++, Rnw, Rmd, Rhtml, Rd)</g:item>
+            <g:item value="*.r, *.R, *.rnw, *.Rnw, *.rmd, *.Rmd, *.rmarkdown, *.Rmarkdown, *.rhtml, *.Rhtml, *.h, *.hpp, *.c, *.cpp">Common R source files (R, C/C++, Rnw, Rmd, Rhtml)</g:item>
             <g:item value="*.r, *.R">R Scripts</g:item>
-            <g:item value="package">Package Source (R/ , src/, tests/)</g:item>
+            <g:item value="packageSource">Package Source (R/ , src/)</g:item>
```

Comment:
I would prefer "Package Sources" or "Package Source Files" as a label here. (@jmcphers do you have an opinion?)

Reply:
"Package Sources" (the simpler the better!)

Reply:
Done!